x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/plotUtils.js

```diff
@@ -75,22 +78,26 @@ export function getPlotValues(
     y: yScale,
     yTickValues,
     XY_MARGIN,
-    XY_HEIGHT,
-    XY_WIDTH: width
+    XY_HEIGHT: height || XY_HEIGHT,
+    XY_WIDTH: width,
+    ...(stacked ? { stackBy: 'y' } : {})
   };
 }
 
 export function SharedPlot({ plotValues, ...props }) {
+  const { XY_HEIGHT: height, XY_MARGIN: margin, XY_WIDTH: width } = plotValues;
```

Comment:
I'm generally not a fan of renaming thing like this. If the names are not good enough, why not fix them at the root? Instead we end up with multiple names for the same thing.